README.md

```diff
@@ -30,3 +30,37 @@ This needs to be running for Cypress to use it.
 This will open a window so you can control which spec files are being ran.
 When you run a spec file test, it will load a new _controlled_ instance of Chrome.
 From there, you will have an interface to see the results from tests, as well as an iframe of your app.
+
+## Tests
+
+To run tests in the interactive mode, start the server and then open Cypress
+
+```
+npm start &
```

Comment:
I prefer not suggesting running a process in the background, simply because not everyone has a lot of sh experience. Is it possible to use your `start-server-and-test` package to do something like: `start-server-and-test start http://localhost:8080 test`?

Reply:
I changed the `npm test` command and updated the README